filebeat/input/filestream/fswatch.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/elastic/beats/v7/filebeat/input/file"
 	loginp "github.com/elastic/beats/v7/filebeat/input/filestream/internal/input-logfile"
 	"github.com/elastic/beats/v7/libbeat/common"
+	filelibbeat "github.com/elastic/beats/v7/libbeat/common/file"
```

Comment:
Could you please rename the alias to `file_helper`?

Reply:
Thanks for reading my PR, I will fix it right away